webpack.config.js

```diff
@@ -68,7 +68,9 @@ const config = (module.exports = {
     // filename: "[name].bundle.js?[chunkhash]",
     filename: "[name].bundle.[contenthash].js",
```

Comment:
Discard this file changes too
As these file changes occurred due to formatting of file

Reply:
This commented code was preserved by Deepak.